### DISCUSSION TITLE
How to change from HTTP to HTTPS?

### DISCUSSION THREAD

**Discussion by myrdrks** [shlinkio/discussions#91] [Help wanted] [answered]
I installed slink via Docker with IS_HTTPS_ENABLED=false but now I need HTTPS.
Where do I set this flag to true? I see no option for this. All my links are http://

#### How Shlink is set-up

Setup via Docker

* Shlink Version: Server: v3.3.2 - Client: v3.7.3
* PHP Version: ?
* How do you serve Shlink: Self-hosted Apache|Self-hosted nginx|Self-hosted openswoole|Docker image
* Database engine used: MySQL|MariaDB|PostgreSQL|MicrosoftSQL|SQLite (x.y.z)

**Comment by acelaya** [comment, marked as answer]
Just switch `-e IS_HTTPS_ENABLED=false` with `-e IS_HTTPS_ENABLED=true`? 🤔

Also to avoid confusions, take into consideration that Shlink does not handle certificates. This env var is just to inform Shlink that you are exposing it via https so that generated links have the proper schema when served from the API.

https://shlink.io/documentation/environment-variables/#general

![Screenshot_20221114-085334](https://user-images.githubusercontent.com/2719332/201604917-58d5d9da-9ca0-406e-b60c-44587836bc5b.png)
